sources/packages/backend/apps/api/src/route-controllers/application/application.students.controller.ts

```diff
@@ -325,12 +339,23 @@ export class ApplicationStudentsController extends BaseController {
    */
   @CheckSinValidation()
   @Patch(":applicationId/draft")
+  @ApiUnprocessableEntityResponse({
+    description: "Invalid program intensity.",
+  })
   @ApiNotFoundResponse({ description: "APPLICATION_DRAFT_NOT_FOUND." })
   async updateDraftApplication(
     @Body() payload: SaveApplicationAPIInDTO,
     @Param("applicationId", ParseIntPipe) applicationId: number,
     @UserToken() studentToken: StudentUserToken,
   ): Promise<void> {
+    const isFulltimeAllowed = this.configService.isFulltimeAllowed;
+    if (
+      !isFulltimeAllowed &&
+      payload.data.howWillYouBeAttendingTheProgram &&
+      payload.data.howWillYouBeAttendingTheProgram === "Full Time"
```

Comment:
Line 354 is not required `payload.data.howWillYouBeAttendingTheProgram === "Full Time"` should be sufficient

Reply:
is the offering intensity not being validated at this point expecting that the submit endpoint will validate it?